loader/src/main/kotlin/net/weavemc/loader/mixin/MixinSandbox.kt

```diff
@@ -103,7 +106,12 @@ class SandboxedMixinLoader(
         }
     }
 
-    override fun loadClass(name: String): Class<*> = findClass(name)
+    override fun loadClass(name: String): Class<*> {
+        if (name.startsWith("me.xtrm.klog.") || name.startsWith("kotlin.")) {
```

Comment:
Could we handle this using the already existing `systemClasses` property instead?

Reply:
Whoops, definitely better